tools/cli.js

```diff
@@ -101,24 +164,37 @@ commandProcessor.exit = () => {
   process.exit();
 };
 
+// str - inputs string
+// separator - string to use as a separator
+// limit - resulting length of output array - 1 (last one is what's left),
+//         if !limit === true => means no limit and split till no more
+//         separators found
+// leaveEmpty - if true multiple separators in sequence will be added as empty
+//              empty string, else they are skipped
+//
+// returns an array of strings
+//
+// the behaviour is as follows:
+//  splits 'str' till limit is bound or no more separators left in 'str'
+//  if leaveEmpty is true then multiple separators in sequence are written in
+//  resulting array as one empty string (''), else they are skipped
+//  and doesn't get counted to limit
 function _split(str, separator, limit, leaveEmpty) {
```

Comment:
@aqrln we originally wrote it like this, as it really is private, i guess i will fix this during decomposition of cli.js in https://github.com/metarhia/JSTP/pull/127